packages/showcases/svelte/.storybook/main.js

```diff
@@ -4,9 +4,13 @@ module.exports = {
     '../stories/components/**/*.stories.{js,jsx,ts,tsx,mdx,svelte}'
   ],
   addons: [
-    '@storybook/addon-svelte-csf',
-    '@storybook/addon-essentials',
     '@storybook/addon-a11y',
+    '@storybook/addon-actions',
+    '@storybook/addon-backgrounds',
+    '@storybook/addon-docs',
+    '@storybook/addon-essentials',
+    '@storybook/addon-svelte-csf',
+    '@storybook/addon-viewport',
```

Comment:
Addons "actions", "backgrounds", "docs" & "viewport" are already present in "essentials" so you don't need to add them.

![CleanShot 2022-03-06 at 22 51 21](https://user-images.githubusercontent.com/9600228/156943607-e24f5752-18b7-4a9a-8a89-42b9ef7ccdcf.png)

Reply:
Yes !
In storybook css showcase contains `@storybook/addon-docs` and others with `@storybook/addon-essentials` it needs to be replace too ?

Reply:
Yes! Feel free to add `@storybook/addon-essentials` in the CSS showcase and replacing all packages "backgrounds" etc.

Reply:
Done :)